### DISCUSSION TITLE
Rüyada uzaylı ziyareti sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#184] [Zeka Sorusu]
Rüyanızda uzay mekiğinize binip aya gidiyorsunuz ve orada isimleri A, B ve C olan üç uzaylıyla tanışıyorsunuz. Tanıştığınız uzaylılardan bir tanesi Merkürlü, bir tanesi Venüslü ve bir tanesi de Marslıdır. Sabah uyandığınızda hangi uzaylının nereli olduğuna ilişkin sadece aşağıda verilenleri hatırlıyorsunuz.
● Eğer C Merkürlü ise B Venüslüdür.
● Eğer C Venüslü ise B Marslıdır.
● Eğer A Marslı ise C Venüslüdür.
● Eğer B Merkürlü değilse A Venüslüdür.
Hatırladıklarınızın ışığında hangi uzaylı nerelidir?

**Comment by Eceg25** [comment]
A Venüslü, B Merkürlü, C Marslıdır.

**Comment by ecedeger** [comment]
C'nin Merkürlü olma olasılığını düşünürsek, B'nin Venüslü olması gerekir. Fakat B Merkürlü olmadığından A'nın da Venüslü olmaı gerekir bu yüzden bu olasılık imkansızdır.
C'nin Venüslü olma olasılığını düşnürsek, B'nin Marslı olması gerekir. Fakat yine B Merkürlü olmadığından A'nın da Venüslü olması gerekir bu yüzden bu olasılık da imkansızdır.
C'nin Marslı olma olasılığını düşünürsek, B ve A'nın ait olduğu gezegenleri bulmamız gerekir. B'nin Venüslü olduğunu düşünürsek Merkürlü olmadığı için A'nın da Venüslü olması gerekir ama bu olasılık imkansızdır. Bu nedenle A Venüslüdür ve B Merkürlüdür.
Yani sonuç olarak A Venüslüdür, B Merkürlüdür, C Marslıdır.
